notary-signer-Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM diogomonica/golang-softhsm2
+FROM dockersecurity/golang-softhsm2
```

Comment:
Since `docker.io/dockersecurity/golang-softhsm2` doesn't exist, `docker-compose up` will fail.  Can we add another instruction to the README to `cd signer; docker build -t dockersecurity/golang-softhsm2 .`?